lib/ansible/utils/role_finder.py

```diff
@@ -0,0 +1,181 @@
+# (c) 2021 Ansible Project
```

Comment:
bikeshed: i would put under lib/ansible/playbook/roles/finder.py

Reply:
CollectionFinder is in utils, so...

Reply:
also going to move that to collections/